### ISSUE TITLE
Missing description of calibration datasets

### ISSUE THREAD

**Issue by jbeuke** [MagneticParticleImaging/OpenMPIData.jl#3]
In https://github.com/MagneticParticleImaging/OpenMPIData.jl/blob/master/docs/src/calibration.md only the 19x19x19 system matrices are described. The other four are not documented.

**Comment by tknopp** [comment]
yes, that is not intended. We could simply say "19x19x19 and 19x19x19" in that field?

**Comment by jbeuke** [comment]
I am not really sure what you mean. The 37x37x37 matrices have a different amount of periods which can be averaged.
